structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,135 @@
+# Summary
+
+This document describes the Quilt Community Collab program, the roles of each member and the associated processes.
+
+# Motivation
+
+The Minecraft community is a complex space, where it can be required for moderators to share intel between communities. The main goal of the Quilt Community Collab program is to facilitate exchange between a selected group of communities and improve user safety.
+
+The server also serves as a coordination server for events and other projects happening in the Minecraft sphere.
+
+# Explanations
+
+The Quilt Community Collab is a private space for Minecraft-centered communities to exchange information and coordinate. The main focus of the exchanges is set on moderation, but it can also be used for events and anything else that may be fitting.
+
+## What Community Collab isn’t
+
+- It isn’t an official affiliation with the Quilt project itself or any other community part of the program
+- It isn’t a place to blindly synchronise punishments between communities
+    - Each community must decide on appropriate moderation action based on facts shown to them and their own rules and policies. Infringing this rule may get your community removed from the Community Collab program.
+
+## Roles inside the server
+
+### Keyholder
+
+The keyholder for this space is the same as for the Quilt Community Team. This role is described in the [RFC 0007 # Keyholders](https://github.com/QuiltMC/rfcs/blob/master/structure/0007-community-team.md#keyholders).
+
+### Managers
+
+Managers are responsible for keeping the collab program up and running.
+
+- Maintain and organise the associated Discord server
+- Actioning changes to the collab-related spaces
+- Organise polls and votes as required by formal processes
+- Help resolve conflicts between communities
+- Stay in touch with any community part of the program
+
+Community Collab Managers are the same as Community Managers of the Quilt Community Team.
+
+### Collaborators
+
+Collaborators are moderators of one or more communities in the program. They have access to the server to the moderation part of the server, such as general moderator chats and case threads. Access is also given to the event category.
+
+### Event Planners
+
+Event Planners are members of the staff team of one or more communities in the program, who focus on event organisation. They have access to the events part of the server.
+
+### Project Roles
+
+Other roles may be given surrounding the different projects happening inside the program. Those roles are informal and will depend on each project.
+
+### Invitees
+
+Invitees are temporary users, with no access to the rest of the server. They only have access to temporary channels required by the process they are visiting for.
```

Comment:
This does not look like the consultant concept I put forward. Essentially, consultants would be collaborators that aren't representing specific communities; they'd have the same access, but would be there to take part in the Collab program without directly representing a group.

I don't see much value in this "Invitees" concept, and I don't believe it was discussed at all.

Reply:
Invitee is a temporary role used for specific process. For example, the members of a server who wish to join CC would get that role while the vote is active. This is different to the consultant role, which currently doesn't exist in this RFC, considering it hasn't ever been used. I'm not opposed to bringing it back though.

Reply:
It was definitely my intention to get that nailed down in this RFC, as we do have potential consultants waiting.